### ISSUE TITLE
[BUG] Mics don't work most of the time

### ISSUE THREAD

**Issue by apex-legends-bugs** [apex-legends-bugs/bugs#7]
**Describe the bug**
Very often, if someone joins your lobby, they cannot communicate via their mic. The speaker icon doesn't light up and their voice cannot be heard. This has been confirmed multiple times with people who said they talk in the game but nothing can be heard. The confirmation was done via text chat or via Xbox party chat.

**To Reproduce**
1. Have someone join your lobby
2. Sometimes they cannot be heard

**Expected behavior**
People should always be heard. Additionally, the speaker icon lighting up should be done via a different system than the party chat service that is failing currently, so that people are at least alerted that someone is trying to talk via the speaker icon lighting up, even if their voice isn't making it through.

**Why could this be happening?**
The voice chat system is not being tested and probably failed and Respawn don't even know about it becuase no one tests this. The service is probably having trouble being connected to by users.

**Additional context**
This is an accessibility issue for people who can't type quickly.

**Workarounds**
Use Xbox Party Chat, PS party chat, or Discord.
